packages/flutter/lib/src/services/image_provider.dart

```diff
@@ -411,6 +411,9 @@ class NetworkImage extends ImageProvider<NetworkImage> {
   /// The scale to place in the [ImageInfo] object of the image.
   final double scale;
 
+  /// The headers that will be used to fech image from network
```

Comment:
fech -> fetch
period at end of sentence

Reply:
Maybe this should be clarified to say HTTP headers, and to point to [HttpClient.get] explicitly and say that it's passed through to that method unmodified?